src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  --hover_color: #00acdc;
+  width: auto;
+  display: flex;
+  align-content: center;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  margin: 10px 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__link {
+  display: inline;
+  height: 15px;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 22.5px 0;
+}
+
+.nav__link:hover {
+  color: var(--hover_color);
+}
+
+.nav__item {
+  display: flex;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.is-active {
+  color: var(--hover_color);
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: relative;
+  top: 19px;
+  height: 4px;
+  background-color: var(--hover_color);
+  border-radius: 8px;
+}
+
+#first__item {
+  margin-left: 0;
+}
+
+#last__item {
+  margin-right: 0;
```

Comment:
Instead of id selectors you can use `.nav__item:first-child` and `.nav__item:last-child`

https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child